### ISSUE TITLE
About new Fill Bucket

### ISSUE THREAD

**Issue by fukhaos** [benrhodes/graffiti-as3-drawing-lib-touch#4]
Thats amazing feature
and works fine
but I have a few questions:
- how I use when I touch (get the touch Point)?
- how disable the brush, when I using the fill (canvas.brush = null, dont works)?
- and how I change the brush to erase ?

``` javascript
var canvas:GraffitiCanvas = new GraffitiCanvas(860, 570);
canvas.x = int((1024-mCanvas.width)/2);
canvas.y = 160;
canvas.overlay = bitmap; 
Starling.current.nativeStage.addChild(canvas);
roundBrush = new RoundBrush(30, 0xff0000, 1);

canvas.brush = roundBrush;
canvas.fill(new Point(100,10), Color.AQUA, true, true, 2);
```

**Comment by benrhodes** [comment]
Yes, it looks like there isn't a really good way to disable a brush to use the Fill method.  The lib needs a fill bucket tool. 

You can change the brush to use ease mode by setting the bitmapCacheRenderMode param to BitmapCacheRenderMode. ERASE in the constructor.
